installer.sh

```diff
@@ -419,6 +423,13 @@ mkdir -p /etc/keylime
 mkdir -p config
 python3 -m keylime.cmd.convert_config --defaults --out config --templates templates
 
+# set the default binding and listening IP about the agent, verifier, and registrar server as 0.0.0.0 when LISTEN_ALL_IPS is set 1
+if [[ "$LISTEN_ALL_IPS" -eq "1" ]] ; then
+    for comp in "agent" "verifier" "registrar"; do
+        sed -i 's/^ip = \"127.0.0.1\"/ip = \"0.0.0.0\"/' config/$comp.conf
```

Comment:
Since you are using single quotes you wouldn't need to escape the double quotes.

Reply:
@stefanberger  OK，I fixed it.